### DISCUSSION TITLE
Arduino Uno 16U2 recognized by the computer like an input 48 MHz audio ?

### DISCUSSION THREAD

**Discussion by ghost** [NicoHood/HID#321] [General]
Hello, 

I got an Arduino Uno,

is it possible to get an HID Audio with a volume control recognized by the computer like an input 48 MHz audio ?

Thanks

**Comment by NicoHood** [comment]
In general this should be possible, however I have not implemented. Reference Implementation can be found in LUFA, but it is not trivial.
